src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesImplementation.OpenSsl.cs

```diff
@@ -62,12 +62,20 @@ private int KeyWrap(ReadOnlySpan<byte> source, Span<byte> destination, int enc)
 
                     IntPtr algorithm = GetKeyWrapAlgorithm(keySizeInBits);
 
-                    return Interop.Crypto.EvpCipherCreate(
+                    SafeEvpCipherCtxHandle ctx = Interop.Crypto.EvpCipherCreate(
```

Comment:
One of these days we should make all LibraryImport's `private` and the internal ones do the error checking like we have recently started doing.